lib/dice_bag/private_key.rb

```diff
@@ -9,32 +8,33 @@ def initialize(key)
       @private_key = key
     end
 
-    def is_valid_private_key?
+    def valid_private_key?
       require 'openssl'
+
       begin
         OpenSSL::PKey::RSA.new @private_key
-        return true  
+        true
       rescue => e
         p e.message
         p e.backtrace
```

Comment:
Not related but I'd rather use `puts` here for clarity. Also these two calls can be merged into one with a line break in between.